src/explorer/LogStreamTreeItem.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ISiteTreeRoot } from 'vscode-azureappservice';
+import { AzureTreeItem } from 'vscode-azureextensionui';
+import { FolderTreeItem } from './FolderTreeItem';
+
+const openLogStreamCommand: string = 'appService.OpenLogStream';
+const connectLogStream: string = 'Connect to log stream...';
```

Comment:
nit: doesn't seem necessary to have a constant when you could just do this below:
```
public commandId: string = 'appService.OpenLogStream';
```